lms/static/styles/lms.scss

```diff
@@ -1,16 +1,22 @@
 // CSS entry point for back-end app
 @use 'normalize.css/normalize';
 
-@use 'variables' as var;
+$color-text: #202020;
+
+// Typography.
+$sans-font-family: 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande',
+  sans-serif !default;
+
+$normal-font-size: 13px;
```

Comment:
👀 - If/when we convert the backend-rendered pages to Tailwind, the default font size should obviously be bumped up to the 16px default.